### PR TITLE
[quickjs][2020.11.08] - New Package

### PR DESCRIPTION
**Pull request by khanhas**

### CODE REVIEW

File: package/quickjs/package [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(quickjs)
+pkgdesc="A small and embeddable Javascript engine"
+url=https://bellard.org/quickjs/
+pkgver=2020.11.08-1
+section=utils
+timestamp=2020-11-08T00:00Z
```
Comment (Eeems) [inline package/quickjs/package]:
```suggestion
timestamp=2020-11-08T13:44Z
```

File: package/quickjs/package [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(quickjs)
+pkgdesc="A small and embeddable Javascript engine"
+url=https://bellard.org/quickjs/
+pkgver=2020.11.08-1
+section=utils
```
Comment (Eeems) [inline package/quickjs/package]:
```suggestion
section=devel
```

File: package/quickjs/package [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+timestamp=2020-11-08T00:00Z
+maintainer="khanhas <xuankhanh963@gmail.com>"
+license=MIT
+
+image=base:v1.1
```
Comment (Eeems) [inline package/quickjs/package]: Any reason you aren't using the latest?
```suggestion
image=base:v1.3.2
```

**Comment by Eeems** [comment]
I wonder if the patch is even needed due to the cross compile setup in the toolchain?

**Comment by khanhas** [comment]
>I wonder if the patch is even needed due to the cross compile setup in the toolchain?

It can cross compile but it only support `WIN32`, `DARWIN` or `x86_64 Linux`. `$CROSS_PREFIX` is left blank and use `gcc` by default if platform doesn't fall in any of them. So I have to patch to force it to use `arm gcc`

**Comment by khanhas** [comment]
Nah, you're right. I could just export those env before running `make`

**Review by raisjn — DISMISSED** [review]
🚀 🦑

File: package/quickjs/package [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+)
+
+build() {
+    patch -u Makefile -i quickjs.patch
+    make qjs.arm
+}
+
+package() {
+    install -d "$pkgdir"/opt/bin
+    install -D -m 755 -t "$pkgdir"/opt/bin/qjs "$srcdir"/qjs.arm
```
Comment (raisjn) [inline package/quickjs/package]: i think this is creating a directory /opt/bin/qjs, should remove the -t as i assume you mean to create `/opt/bin/qjs` (renaming qjs.arm in the process)

Comment (khanhas) [reply package/quickjs/package]: Oh heck, you're right.

**Review by raisjn — APPROVED** [review]

**Pull request merged**